File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {

        unordered_set<int> datamap;

        for(int num: nums){
            if(datamap.count(num)){
                return true;
            }

            datamap.insert(num);
        }

        return false;
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
       boolean isExist = false;
       for(int i:nums){
           System.out.println(i);
           if(map.containsKey(i)){
               isExist = true;
               break;
           }else{
             map.put(i,1);
           }
          
       }
       return isExist; 
    }
}